src/client/components/pages/Courses/CourseInstanceTable.tsx

```diff
@@ -117,10 +117,6 @@ const CourseInstanceTable: FunctionComponent<CourseInstanceTableProps> = ({
     ({ columnGroup }): boolean => (
       columnGroup === COURSE_TABLE_COLUMN_GROUP.META)
   );
-  const firstEnrollmentField = fallColumns
-    .findIndex(({ viewColumn }): boolean => (
-      viewColumn === COURSE_TABLE_COLUMN.ENROLLMENT
```

Comment:
Since we're deleting all of these, `COURSE_TABLE_COLUMN` no longer needs to be imported on line 15

Reply:
Good point. Fixed in 2daf1ca6